AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/MAX/Native Ads/ALMAXManualNativeAdViewController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  ALMAXManualNativeAdViewController.swift
+//  AppLovin MAX Demo App - Swift
+//
+//  Created by Billy Hu on 1/20/22.
+//  Copyright © 2022 AppLovin. All rights reserved.
+//
+
+import UIKit
+import Adjust
+import AppLovinSDK
+
+class ALMAXManualNativeAdViewController: ALBaseAdViewController, MAAdRevenueDelegate
```

Comment:
Can we put `MAAdRevenueDelegate` into an extension, and put that extension at bottom of file. I'm pretty sure our other Swift examples do that as well.